From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Tue, 17 Dec 2019 19:12:36 +0100
Subject: ANDROID: export mm_trace_rss_stat as gpl-only

The commit with change-id: Icb56b0c760780c3f9081f82d921406d17f4f1c08
exported mm_trace_rss_stat which is great, but as this change is not
upstream it needs to be exported as gpl-only.

Cc: Yongqin Liu <yongqin.liu@linaro.org>
Cc: Alistair Delva <adelva@google.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: Ib72cf26a7824415b4beb5133c2cb477ec23d65c1
---
 mm/memory.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/memory.c b/mm/memory.c
--- a/mm/memory.c
+++ b/mm/memory.c
@@ -157,7 +157,7 @@ void mm_trace_rss_stat(struct mm_struct *mm, int member, long count)
 {
 	trace_rss_stat(mm, member, count);
 }
-EXPORT_SYMBOL(mm_trace_rss_stat);
+EXPORT_SYMBOL_GPL(mm_trace_rss_stat);
 
 #if defined(SPLIT_RSS_COUNTING)
 
